dev-python/neovim-python-client/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST neovim-python-client-0.1.13.tar.gz 41413 SHA256 b2f78226d0b3c57d1e53e26828f5fa5665c0655725fbadd43aea87f5ce93ef3b SHA512 ecef42601632551c98f8de019bfdf4dbff3ce1f0e1b6943b78a506b710e5810f6a7e611adc09435992b97288ff365a416870a74202a5c68236b5daeb51d15c8b WHIRLPOOL a7a2193d193797fd406f7a822528773f101c1e5c015c1d7208de042373b382fe09224afbf000158797bc5de96384625d8e1fcdfbc3e7101667ff55382c7a8cca
-DIST neovim-python-client-0.1.9.tar.gz 40599 SHA256 9e6250cad242fb79c07346cd292b055f4f334635a42bff3437131205bb09e0a8 SHA512 043429407d140b37004d5344f9244c8cb5a37e2b5d450454e6535bece7fb05f037e5eff2a63431fad55ab2470372c4a17e0a608a94af65a2f400f2158b736ce4 WHIRLPOOL eb8b54bdc2c5f455862026a39f76bb32f59b3225c17ff2abf2751670f22ed81b053417aef0e95d0fad6fd31df58867024967c44ba48b83b63c70e1c1977de23b
```

Comment:
the checksum of 0.1.9 should not be changed without verification, if we just bump to 0.2.0

Reply:
It didn't, sha512 matches. Sha256 is dropped, and Blake2 added, but the file didn't change.

Reply:
Should I amend this to keep the hashes of 0.1.9 as they were originally?  This was just from a `repoman manifest`.  I had assumed we'd want to update them, but I can revert that if desired.